src/templates/addons/terraformCloud/index.ts

```diff
@@ -0,0 +1,88 @@
+import dedent = require('dedent');
+import { prompt } from 'inquirer';
+
+import { GeneralOptions } from '@/commands/generate';
+import { injectToFile } from '@/helpers/file';
+import {
+  INFRA_BASE_MAIN_PATH,
+  INFRA_SHARED_MAIN_PATH,
+} from '@/templates/core/constants';
+
+type TerraformCloudOptions = {
+  terraformCloudEnabled: boolean;
+  terraformCloudOrganization: string;
+  terraformCloudWorkspace: string;
+};
+
+const getTerraformCloudOptions = async (
+  projectName: string
+): Promise<TerraformCloudOptions> => {
+  const terraformCloudPrompt = await prompt([
+    {
+      type: 'confirm',
+      name: 'terraformCloudEnabled',
+      message: 'Would you like to enable Terraform Cloud?',
+      default: false,
+    },
+  ]);
+
+  if (terraformCloudPrompt.terraformCloudEnabled) {
+    const terraformCloudOptions = await prompt([
+      {
+        type: 'input',
+        name: 'terraformCloudOrganization',
+        message: 'What is your Terraform Cloud organization?',
+        default: 'nimble',
+      },
+      {
+        type: 'input',
+        name: 'terraformCloudWorkspace',
+        message: 'What is your Terraform Cloud workspace?',
+        default: projectName,
```

Comment:
this is not enough since the generated project requires 2 workspace
- 1 workspace for the `shared` directory (ECR, IAM users & groups, ...)
- 1 workspace for the `core` directory (ECS, RDS, ...)

How about changing the first message
> Would you like to enable Terraform Cloud?

to
> Would you like to enable Terraform Cloud? 
If yes, the generated project will require 2 workspaces. One for the "shared" services, such as ERC, that will be used across all environments. The other workspace will be for the "core" services that will be created independently for each environment.

Then the CLI will ask the user to provide the names for the 2 workspaces.

Reply:
It makes sense. Added in https://github.com/nimblehq/infrastructure-templates/pull/206/commits/ca0fad8e55810277cc41fff9c8b16aba850f52be
Please note that I still use `base` for the `core`. But I agree that the `core` makes more sense, so I will have another task to rename it 🙏